default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
So these are the actual differences in how we currently generate UWP projects and how this PR generates them.

1) Wrong tool version.

I see that currently we generate this:
```
<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
... but with this PR Protobuild generates this:
```
<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
This is probably not a good thing and it should be fixed... in fact it might be the main reason this was failing to begin with.

2) Not including `MonoGame.Framework.WindowsUAP.project.json`:

This one might actually not matter, but we currently include this like so:
```
  <ItemGroup>
    <None Include="MonoGame.Framework.WindowsUAP.project.json">
      <Platforms>WindowsUniversal</Platforms>
    </None>
  </ItemGroup>
```
Likely this doesn't really change anything....  but it is a difference.

3) Crazy assembly references:

In the current build the UWP references look like this:
```
    <Reference Include="System" />
    <Reference Include="System.Runtime.Serialization" />
    <Reference Include="SharpDX">
      <HintPath>..\ThirdParty\Dependencies\SharpDX\Windows%20UAP\SharpDX.dll</HintPath>
    </Reference>
    <Reference Include="SharpDX.Direct2D1">
      <HintPath>..\ThirdParty\Dependencies\SharpDX\Windows%20UAP\SharpDX.Direct2D1.dll</HintPath>
    </Reference>
    <Reference Include="SharpDX.Direct3D11">
      <HintPath>..\ThirdParty\Dependencies\SharpDX\Windows%20UAP\SharpDX.Direct3D11.dll</HintPath>
    </Reference>
    <Reference Include="SharpDX.DXGI">
      <HintPath>..\ThirdParty\Dependencies\SharpDX\Windows%20UAP\SharpDX.DXGI.dll</HintPath>
    </Reference>
    <Reference Include="SharpDX.MediaFoundation">
      <HintPath>..\ThirdParty\Dependencies\SharpDX\Windows%20UAP\SharpDX.MediaFoundation.dll</HintPath>
    </Reference>
    <Reference Include="SharpDX.XAudio2">
      <HintPath>..\ThirdParty\Dependencies\SharpDX\Windows%20UAP\SharpDX.XAudio2.dll</HintPath>
    </Reference>
```
But with this PR it has become a monstrosity:
```
rics.Vectors">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Numerics.Vectors\WindowsUniversal\ref\dotnet\System.Numerics.Vectors.dll</HintPath>
    </Reference>
    <Reference Include="System.ObjectModel">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.ObjectModel\WindowsUniversal\ref\dotnet\System.ObjectModel.dll</HintPath>
    </Reference>
    <Reference Include="System.Reflection">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Reflection\WindowsUniversal\ref\dotnet\System.Reflection.dll</HintPath>
    </Reference>
    <Reference Include="System.Reflection.DispatchProxy">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Reflection.DispatchProxy\WindowsUniversal\ref\dotnet\System.Reflection.DispatchProxy.dll</HintPath>
    </Reference>
    <Reference Include="System.Reflection.Extensions">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Reflection.Extensions\WindowsUniversal\ref\dotnet\System.Reflection.Extensions.dll</HintPath>
    </Reference>
    <Reference Include="System.Reflection.Metadata">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Reflection.Metadata\WindowsUniversal\lib\dotnet\System.Reflection.Metadata.dll</HintPath>
    </Reference>
    <Reference Include="System.Reflection.Primitives">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Reflection.Primitives\WindowsUniversal\ref\dotnet\System.Reflection.Primitives.dll</HintPath>
    </Reference>
    <Reference Include="System.Reflection.TypeExtensions">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Reflection.TypeExtensions\WindowsUniversal\ref\dotnet\System.Reflection.TypeExtensions.dll</HintPath>
    </Reference>
    <Reference Include="System.Resources.ResourceManager">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Resources.ResourceManager\WindowsUniversal\ref\dotnet\System.Resources.ResourceManager.dll</HintPath>
    </Reference>
    <Reference Include="System.Runtime">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Runtime\WindowsUniversal\ref\dotnet\System.Runtime.dll</HintPath>
    </Reference>
    <Reference Include="System.Runtime.Extensions">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Runtime.Extensions\WindowsUniversal\ref\dotnet\System.Runtime.Extensions.dll</HintPath>
    </Reference>
    <Reference Include="System.Runtime.Handles">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Runtime.Handles\WindowsUniversal\ref\dotnet\System.Runtime.Handles.dll</HintPath>
    </Reference>
    <Reference Include="System.Runtime.InteropServices">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Runtime.InteropServices\WindowsUniversal\ref\dotnet\System.Runtime.InteropServices.dll</HintPath>
    </Reference>
    <Reference Include="System.Runtime.Numerics">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Runtime.Numerics\WindowsUniversal\ref\dotnet\System.Runtime.Numerics.dll</HintPath>
    </Reference>
    <Reference Include="System.Security.Claims">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Security.Claims\WindowsUniversal\ref\dotnet\System.Security.Claims.dll</HintPath>
    </Reference>
    <Reference Include="System.Security.Principal">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Security.Principal\WindowsUniversal\ref\dotnet\System.Security.Principal.dll</HintPath>
    </Reference>
    <Reference Include="System.Text.Encoding">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Text.Encoding\WindowsUniversal\ref\dotnet\System.Text.Encoding.dll</HintPath>
    </Reference>
    <Reference Include="System.Text.Encoding.Extensions">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Text.Encoding.Extensions\WindowsUniversal\ref\dotnet\System.Text.Encoding.Extensions.dll</HintPath>
    </Reference>
    <Reference Include="System.Text.RegularExpressions">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Text.RegularExpressions\WindowsUniversal\ref\dotnet\System.Text.RegularExpressions.dll</HintPath>
    </Reference>
    <Reference Include="System.Threading">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Threading\WindowsUniversal\ref\dotnet\System.Threading.dll</HintPath>
    </Reference>
    <Reference Include="System.Threading.Tasks">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Threading.Tasks\WindowsUniversal\ref\dotnet\System.Threading.Tasks.dll</HintPath>
    </Reference>
    <Reference Include="System.Threading.Tasks.Dataflow">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Threading.Tasks.Dataflow\WindowsUniversal\lib\dotnet\System.Threading.Tasks.Dataflow.dll</HintPath>
    </Reference>
    <Reference Include="System.Threading.Tasks.Parallel">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Threading.Tasks.Parallel\WindowsUniversal\ref\dotnet\System.Threading.Tasks.Parallel.dll</HintPath>
    </Reference>
    <Reference Include="System.Threading.Timer">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Threading.Timer\WindowsUniversal\ref\dotnet\System.Threading.Timer.dll</HintPath>
    </Reference>
    <Reference Include="System.Xml.ReaderWriter">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Xml.ReaderWriter\WindowsUniversal\ref\dotnet\System.Xml.ReaderWriter.dll</HintPath>
    </Reference>
    <Reference Include="System.Xml.XDocument">
      <HintPath>..\Microsoft.NETCore.UniversalWindowsPlatform\WindowsUniversal\Microsoft.NETCore\WindowsUniversal\System.Xml.XDocument\WindowsUniversal\ref\dotnet\System.Xml.XDocument.dll</HintPath>
    </Reference>
```
Another reason Protobuild has to stop trying to fix UWP itself.

Also found this new thing as of this PR:
```
  <PropertyGroup>
    <_PostBuildHookTimestamp>@(IntermediateAssembly-&gt;'%(FullPath).timestamp')</_PostBuildHookTimestamp>
  </PropertyGroup>
  <Target Name="PostBuildHooks" Inputs="@(IntermediateAssembly);@(ReferencePath)" Outputs="@(IntermediateAssembly);$(_PostBuildHookTimestamp)" AfterTargets="CoreCompile" BeforeTargets="AfterCompile">
    <Touch Files="$(_PostBuildHookTimestamp)" AlwaysCreate="True" />
  </Target>
```
Is that something new in VS2015?  Or is that some Protobuild feature?  Or something else?